include/stc/priv/utf8_prv.c

```diff
@@ -34,11 +34,11 @@ const uint8_t utf8_dtab[] = {
    7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,  7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,
    8,8,2,2,2,2,2,2,2,2,2,2,2,2,2,2,  2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,
   10,3,3,3,3,3,3,3,3,3,3,3,3,4,3,3, 11,6,6,6,5,8,8,8,8,8,8,8,8,8,8,8,
-   0,12,24,36,60,96,84,12,12,12,48,72, 12,12,12,12,12,12,12,12,12,12,12,12,
```

Comment:
Change 12 to -1, so we can simply write `state < 0`, which keeps the performance baseline.

After -1, the decode state machine is messed up, further calls will just return junk values.